sdk/communication/AzureCommunication/Source/Authentication/CommunicationAuthenticationPolicy.swift

```diff
@@ -28,45 +28,33 @@
 import AzureCore
 #endif
 import Foundation
-/**
- The Azure Communication Services authentication policy.
- */
-public class CommunicationUserCredentialPolicy: Authenticating {
-    public var next: PipelineStage?
 
+public class CommunicationPolicyTokenCredential: TokenCredential {    
```

Comment:
Missed comment in the class CommunicationPolicyTokenCredential

Reply:
Added comment block